templates/medical_questions_pairs/templates.yaml

```diff
@@ -0,0 +1,129 @@
+dataset: medical_questions_pairs
+templates:
+  18c92f97-0655-4f67-aca1-69f8e4fbb11e: !Template
+    id: 18c92f97-0655-4f67-aca1-69f8e4fbb11e
+    jinja: 'In the context of healthcare questionnaires, it is often necessary to
+      find out if two questions are paraphrases of one another. Given the following
+      question:
+
+
+      {% if label == 0 or label == 1 %}
+
+
+      Question 1: {{question_1}}
+
+
+      What inference can be made about the next question?
+
+
+      Question 2: {{question_2}}
+
+
+      {% endif %}
+
+
+      |||
+
+
+      {% if label == 0 %}
+
+
+      This question is not a paraphrase of the previous question.
+
+
+      {% elif label == 1 %}
+
+
+      This question is a paraphrase of the previous question.
+
+
+      {% endif %}'
+    name: basic_v5
+    reference: ''
+    task_template: false
+  316f2ff7-45f8-4997-9c5f-dfe5fb7f9808: !Template
+    id: 316f2ff7-45f8-4997-9c5f-dfe5fb7f9808
+    jinja: "{% if label == 0 or label == 1 %}\n\nQuestion 1: {{question_1}}\n\nand\n\
+      \nQuestion 2: {{question_2}}\n\n{% endif %}\n\nIs it True or False that the\
+      \ two questions above are paraphrases of each other?\n\n|||\n\n {{[\"False\"\
+      , \"True\"][label]}} "
+    name: basic_v6
+    reference: ''
+    task_template: false
+  535ed335-de9d-41df-a026-28487c832bfa: !Template
+    id: 535ed335-de9d-41df-a026-28487c832bfa
+    jinja: 'Question: In the context of healthcare do the following questions mean
+      the same thing?
+
+
+      Question 1: {{question_1}}
+
+
+      Question 2: {{question_2}}
+
+
+      |||
+
+
+      {{["No they do not mean the same thing.", "They mean the same thing."][label]}}'
+    name: basic_v2
+    reference: template about question pattern
+    task_template: true
+  7be2b267-8d5c-466b-9fd4-1fbbae442938: !Template
+    id: 7be2b267-8d5c-466b-9fd4-1fbbae442938
+    jinja: '{% if label == 0 or label == 1 %}
```

Comment:
i don't understand this condition. there can't be other values for label than 0 and 1 so this condition is not necessary

Reply:
found 5 occurences of this. same comment